lib/Models/WebProcessingServiceCatalogItem.js

```diff
@@ -339,4 +275,32 @@ function formatOutputValue(title, value) {
     }, '');
 }
 
+function getGeoJson(catalogItem) {
+    if (!defined(catalogItem.parameters) || !defined(catalogItem.parameterValues)) {
+        return;
```

Comment:
Actually, it might be better to `return when(undefined);`.  That way the function always returns a promise, and that promise may or may not resolve to undefined, rather than needing to check for undefined in two places.  You could remove the `when` at the call site.